tests/test_cli.py

```diff
@@ -46,32 +48,98 @@ def assert_expected_file(self, result, output_path):
+    )
+    def test_schema_convert_cli(self, runner, output_path, output_type):
+        model = "tests/data/example.model.csv"
+        label_type = "class_label"
+        expected = 0
 
-        output_path = helpers.get_data_path("example.model.jsonld")
+        resultOne = runner.invoke(schema, ["convert", model])
 
-        label_type = "class_label"
+        assert resultOne.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
+
+        resultTwo = runner.invoke(
+            schema, ["convert", model, "--output_path", output_path]
+        )
+
+        assert resultTwo.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
+
+        resultThree = runner.invoke(
+            schema, ["convert", model, "--output_type", output_type]
+        )
+
+        assert resultThree.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
+
+        resultFour = runner.invoke(
+            schema,
+            [
+                "convert",
+                model,
+                "--output_type",
+                output_type,
+                "--output_jsonld",
+                output_path,
+            ],
+        )
+
+        assert resultFour.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
 
         result = runner.invoke(
             schema,
             [
                 "convert",
-                data_model_csv_path,
+                model,
                 "--output_jsonld",
                 output_path,
                 "--data_model_labels",
                 label_type,
             ],
         )
 
-        assert result.exit_code == 0
+        assert result.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
 
-        expected_substr = (
-            "The Data Model was created and saved to " f"'{output_path}' location."
+        resultFive = runner.invoke(
+            schema,
+            [
+                "convert",
+                model,
+                "--output_jsonld",
+                "tests/data/example.model.pickle",
+                "--output_path",
+                "tests/data/example.model.pickle",
+            ],
         )
 
-        assert expected_substr in result.output
+        assert resultFive.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
+
+        resultSix = runner.invoke(
+            schema, ["convert", model, "--output_jsonld", "", "--output_path", ""]
+        )
+
+        assert resultSix.exit_code == expected
+        # check output_path file is created then remove it
+        assert os.path.exists(output_path)
```

Comment:
I think I added this comment in the other PR, but this could be split up into separate test cases to clearly define each of the different behaviors.  OR.  make use of `pytest.parametrize`

Reply:
+ 1 to Tom's suggestion. Please split this up so that it is more clear what has been tested.